src/styles/calendar.scss

```
@@ -0,0 +1,57 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $boxGap;
+  width: $daysCount * $boxSize + ($daysCount - 1) * $boxGap;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  & > div {
+    background-color: #eee;
+    border: 1px solid;
+    width: $boxSize;
+    height: $boxSize;
+    text-align: center;
+    line-height: $boxSize;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i}) {
+        &:before {
+          content: "#{$i}";
+        }
+      }
+    }
+  }
+
+  $days:
+    0 "mon",
+    1 "tue",
+    2 "wed",
+    3 "thu",
+    4 "fri",
+    5 "sat",
+    6 "sun";
+
+  @each $number, $day in $days {
+    &-#{$day} > div:first-child {
+      margin-left: ($boxSize + $boxGap) * $number;
```

Так має краще працювати, бо поки верхній ряд з'їжджає на 1 піксель
```suggestion
      margin-left: $boxSize * $number + $number;
```